src/items.js

```diff
@@ -56,15 +56,34 @@ const appCreator = () => {
 
       const applyButton = components.newButton({
         classNames: "btn btn-outline-success",
-        onClick: () => applyKey(item.id),
+        onClick: () => {
+          const textArea = document.getElementById(`token-${item.id}`);
+          textArea.disabled = true;
```

Comment:
Instead of disable, shouldn't it hide the token?
If the user is coming from the "view", it should make sense to keep it (which doesn't seem to me to be the common case).
But, if the user just presses the set button, it doesn't make sense to keep showing the value afterward.

Let me know what do you think.

Reply:
We have specific code for this behavior - when clicking apply to enter in view mode and inside the dropdown menu change to hide value (in view option).
We did this to reassure the user changes were made according to his new input. 

👁️ Maybe we can rethink this, if the user really need to have that view mode after setting a new value. But I think it would out of scope for this PR, if you agree. Let's discuss this better offline and if we come to a different conclusion, create a new task. WDYT?

Reply:
Perfect. We have a plan. Thank you @Marianarodrigues7 🙇🏻